src/nls/ko/strings.js

```diff
@@ -92,26 +92,26 @@ define({
 
     // Application error strings
     "ERROR_IN_BROWSER_TITLE": "{APP_NAME} 아직 브라우저에서 실행되지 않습니다.",
-    "ERROR_IN_BROWSER": "{APP_NAME}는 HTML로 구축되어 있지만 데스크톱 응용 프로그램으로 실행하여 로컬 파일을 편집 할 수 있습니다. {APP_NAME}을 실행하기 위해 <b>github.com / adobe / brackets-shell</b> 저장소 응용 프로그램 셸을 사용하십시오.",
+    "ERROR_IN_BROWSER": "{APP_NAME}은(는) HTML로 구축되어 있지만 데스크톱 응용 프로그램으로 실행하여 로컬 파일을 편집 할 수 있습니다. {APP_NAME}을 실행하기 위해 <b>github.com / adobe / brackets-shell</b> 저장소 응용 프로그램 셸을 사용하십시오.",
```

Comment:
Don't use both two postpositions. `{APP_NAME}` will be replaced with `"Brackets"`.